lib/chef/provider/mount/aix.rb

```diff
@@ -149,6 +180,18 @@ def enable_fs
           end
         end
 
+        def mount_options_unchanged?
+          current_resource_options = @current_resource.options
+          current_resource_options.pop
```

Comment:
Hi @btm ,
 
It pop the extra `"rw"` which gets appended for options comming from file `/etc/filesystems`
 & which in return does not match with the options we are passing from the recipe.
Hence I am removing the last entry which is `"rw"` from the `current_resource_options` so it can match the exact entries from the `/etc/filesystems` file.